.travis.yml

```diff
@@ -7,23 +7,45 @@ os:
     - linux
     - osx
 addons:
-        apt:
-                packages:
-                        - rpm
+    apt:
+        packages:
+            - rpm
+            - libnl-3-200
+            - libnl-3-dev
+            - libnl-route-3-200
+            - libnl-route-3-dev
+    ssh_known_hosts:
+        - flatbed.openfabrics.org
+        - git.kernel.org
+
+env:
+    global:
+        - CPPFLAGS=-I$HOME/include
+        - LDFLAGS=-L$HOME/lib
+        - LD_LIBRARY_PATH=$HOME/lib
+        - PROVIDERS="--enable-verbs --enable-usnic --enable-sockets"
+
+# install dependencies for the verbs and usnic providers
+before_install:
+        - git clone https://git.kernel.org/pub/scm/libs/infiniband/libibverbs.git
+        - cd libibverbs && ./autogen.sh && ./configure --prefix=$HOME && make && make install && cd ..
+        - git clone git://flatbed.openfabrics.org/~shefty/librdmacm.git
```

Comment:
I think we can use www.openfabrics.org here, rather than naming flatbed.  OFA tends to replace their servers every couple of years and rename them, but I've always just continued using the common names.